src/React/SyntheticEvent.purs

```diff
@@ -193,3 +226,180 @@ type SyntheticWheelEvent' r
     , deltaZ :: Number
     | r
     )
+
+type SyntheticAnimationTransitionEvent' r
+  = ( pseudoElement :: String
+    , elapsedTime :: Number
+    | r
+    )
+
+type SyntheticFocusMouseEvent'  r
+  = ( relatedTarget :: NativeEventTarget
+    | r
+    )
+
+type SyntheticKeyboardMouseTouchEvent' r
+  = ( altKey :: Boolean
+    , ctrlKey :: Boolean
+    , metaKey :: Boolean
+    , shiftKey :: Boolean
+    | r
+    )
+
+bubbles :: forall eff r. SyntheticEvent_ (SyntheticEvent' r) -> Eff eff Boolean
+bubbles = get (SProxy :: SProxy "bubbles")
+
+cancelable :: forall eff r. SyntheticEvent_ (SyntheticEvent' r) -> Eff eff Boolean
+cancelable = get (SProxy :: SProxy "cancelable")
+
+currentTarget :: forall eff r. SyntheticEvent_ (SyntheticEvent' r) -> Eff eff NativeEventTarget
+currentTarget = get (SProxy :: SProxy "currentTarget")
+
+defaultPrevented :: forall eff r. SyntheticEvent_ (SyntheticEvent' r) -> Eff eff Boolean
+defaultPrevented = get (SProxy :: SProxy "defaultPrevented")
+
+eventPhase :: forall eff r. SyntheticEvent_ (SyntheticEvent' r) -> Eff eff Number
+eventPhase = get (SProxy :: SProxy "eventPhase")
+
+isTrusted :: forall eff r. SyntheticEvent_ (SyntheticEvent' r) -> Eff eff Boolean
+isTrusted = get (SProxy :: SProxy "isTrusted")
+
+nativeEvent :: forall eff r. SyntheticEvent_ (SyntheticEvent' r) -> Eff eff NativeEvent
+nativeEvent = get (SProxy :: SProxy "nativeEvent")
+
+target :: forall eff r. SyntheticEvent_ (SyntheticEvent' r) -> Eff eff NativeEventTarget
+target = get (SProxy :: SProxy "target")
+
+timeStamp :: forall eff r. SyntheticEvent_ (SyntheticEvent' r) -> Eff eff Number
+timeStamp = get (SProxy :: SProxy "timeStamp")
+
+type_ :: forall eff r. SyntheticEvent_ (SyntheticEvent' r) -> Eff eff String
+type_ = get (SProxy :: SProxy "type")
+
+animationName :: forall eff r. SyntheticEvent_ (SyntheticAnimationEvent' r) -> Eff eff String
+animationName = get (SProxy :: SProxy "animationName")
+
+clipboardData :: forall eff r. SyntheticEvent_ (SyntheticClipboardEvent' r) -> Eff eff NativeDataTransfer
```

Comment:
Additionally, I think that accessors could be polymorphic.

```purescript
clipboardData :: forall eff r. SyntheticEvent_ (clipboardData :: NativeDataTransfer | r) -> Eff eff NativeDataTransfer
```

Reply:
Looks good.